src/server/api/qrcode.ts

```diff
@@ -0,0 +1,24 @@
+import Express from 'express'
+
+import createGoQrCode from '../util/qrcode'
+import ImageFormat from '../../shared/util/imageFormat'
+
+const router = Express.Router()
+
+router.post('/', (req, res) => {
+  const url = req.body.url as string
+  const format = req.body.format as ImageFormat
```

Comment:
What happens when `format` is a string that is outside of the possible enum values?

Reply:
Thanks for picking this up. This is now fixed with an object value check